src/main/kotlin/paladin/discover/models/monitoring/changeEvent/JsonChangeEventHandler.kt

```
@@ -0,0 +1,167 @@
+            after = after,
+            source = source,
+            // Easy access fields from the payload
+            timestamp = payload.get("ts_ms")?.asLong(),
+            table = source?.get("table")?.toString(),
+        )
+    }
+
+    private fun generateJsonNode(value: String): JsonNode {
+        try {
+            return objectMapper.readTree(value)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)